README.md

```diff
@@ -1,4 +1,3 @@
-[![Build Status](https://travis-ci.org/google/renameio.svg?branch=master)](https://travis-ci.org/google/renameio)
```

Comment:
Can you put the GitHub actions CI badge here instead?

Reply:
Done. I also added a commit to update the doc link from `godoc.org` to `pkg.go.dev`.